### PR TITLE
add packit

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by ehelms** [comment]
Cool idea!

**Comment by evgeni** [comment]
note from team discussion: can packit be triggered on demand (comment or so), instead of always building

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  create-archive:
+    - bash -c "cd foreman-installer && bundle config set --local path vendor/bundle"
+    - bash -c "cd foreman-installer && bundle config set --local without development:test"
+    - bash -c "cd foreman-installer && bundle install"
+    - bash -c "cd foreman-installer && bundle exec rake pkg:generate_source"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    manual_trigger: true
```
Comment (evgeni) [inline .packit.yaml:34]: @ekohl @ehelms a tiny bolean making it all just work like we want :)

**Review by ehelms — APPROVED** [review]

File: .packit.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: foreman-installer.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - foreman-installer.spec
+    - .packit.yaml
```
Comment (ekohl) [inline .packit.yaml:9]: Please keep yamllint happy :)
```suggestion
  - foreman-installer.spec
  - .packit.yaml
```

**Review by ekohl — COMMENTED** [review]
Nit, but otherwise :+1:

**Pull request merged**